cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/util/hbase/HBaseDDLExecutorFactory.java

```diff
@@ -88,4 +90,9 @@ protected String getHBase10CHD550ClassName() {
   protected String getHBase12CHD570ClassName() {
     return "co.cask.cdap.data2.util.hbase.DefaultHBase12CDH570DDLExecutor";
   }
+
+  @Override
+  protected CConfiguration getCConfiguration() {
```

Comment:
This is to be called in `HBaseVersionSpecificFactory.get()` but `HBaseVersionSpecificFactory` is an abstract class, so I'm not sure how to pass cConf to it